### PR TITLE
Use hatchling as build backend

### PR DESCRIPTION
**Pull request by hoechenberger**
To fix editable installs. We already switched over MNE-Python, MNE-BIDS, and MNE-BIDS-Pipeline.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -1,72 +1,68 @@
...
 readme = "README.md"
-requires-python = ">=3.8"
-license = {file = "LICENSE"}
+requires-python = ">=3.9"
+license = { file = "LICENSE" }
 keywords = ["science", "neuroscience", "psychology"]
-authors = [
-  {name = "Martin Schulz", email = "dev@earthman-music.de"},
-]
+authors = [{ name = "Martin Schulz", email = "dev@earthman-music.de" }]
```
Comment (marsipu) [inline pyproject.toml:12]:
```suggestion
authors = [{ name = "Martin Schulz", email = "dev@mgschulz.de" }]
```
I have a new e-mail-adress for software-development. Would you kindly change that for me?

**Review by marsipu — DISMISSED** [review]

File: pyproject.toml (hunk 1 above)
Comment (marsipu) [reply pyproject.toml]: Thank you @hoechenberger

**Comment by marsipu** [comment]
I don't have experience with hatchling yet so I probably better leave approval to others.
Thank you @hoechenberger

**Comment by hoechenberger** [comment]
Yes let's wait for @larsoner's feedback / thoughts

**Pull request merged**